internal/test/scorecard/common_utils.go

```diff
@@ -366,6 +366,11 @@ func waitTillCryostatReady(base *url.URL, resources *TestResources) error {
 			return false, fmt.Errorf("API request failed with status code %d: %s", resp.StatusCode, ReadError(resp))
 		}
 
+		if resp.StatusCode == http.StatusNoContent {
```

Comment:
I think this func `waitTillCryostatReady`  is meant to query Cryostat health only. We probably can refactor some common code and create a health check function for report. What do you think?